protocol/vm/vm_test.go

```diff
@@ -9,8 +9,6 @@ import (
 	"testing/quick"
 
 	"chain/errors"
-	"chain/protocol/bc"
-	"chain/protocol/validation"
 	. "chain/protocol/vm"
```

Comment:
It can, but doing that right will involve some code changes (such as using virtualMachine instead of VirtualMachine) that would obscure what we're doing here, and it's relatively easy, so I'd rather do it for all the test files together in a separate "remove export_test.go" PR.